### DISCUSSION TITLE
How do I disable notify and notifier?

### DISCUSSION THREAD

**Discussion by blancpain** [folke/snacks.nvim#41] [Q&A] [answered]
I am using LazyVim - since notify is not coming from nvim-notify anymore(?) how can I disable it? Passing enabled = false in Snacks opts doesn't seem to do the trick.

**Comment by folke** [comment]
Check the snacks.nvim docs?

**Reply by blancpain** [reply]
I did, going with the example config, I have the following setup atm that doesn't disable it:

```lua

return {
  "folke/snacks.nvim",
  priority = 1000,
  lazy = false,
  ---@type snacks.Config
  -- TODO: figure out how to disable notify
  opts = {
    notifier = {
      enabled = false,
    },
  },
}

```

Maybe I am missing something obvious.

**Reply by folke** [reply]
ok, I'll look into it.

**Reply by folke** [reply, marked as answer]
The problem is that notifier was disabled, but noice would still use it.
Just fixed this on the noice side.

**Reply by blancpain** [reply]
Sweet - thanks for the quick fix! Appreciate all your work!
